Story/Extinction.cs

```diff
@@ -0,0 +1,58 @@
+//cs_include Scripts/CoreBots.cs
+//cs_include Scripts/CoreStory.cs
+using Skua.Core.Interfaces;
+
+public class Extinction
```

Comment:
Every file of a story should be included in 0AllStories once it is created so it's not forgotten later
check any other storyfile and thier respective include in 0AllStories following alphapectical order of the filing 

1- //cs_include [scriptPath] 
2- instantiating a class object of it
3- calling storymethod of the class